src/lava/lib/optimization/solvers/generic/nebm/process.py

```diff
@@ -127,7 +127,7 @@ def __init__(
 
         self.refract_counter = Var(
             shape=shape,
-            init=(refract or 0)
+            init=refract
```

Comment:
Please note: The following version of the QUBO solver will fully remove the 'refract' parameter. Thus, I would prefer to have 
'init=0'
here already now. 
And you should start using refract=0 in general, and only set refract_scaling.

Reply:
Point noted.

Reply:
But is there any reason for this change here? If not, I would prefer not having it. Will just cause an unnecessary merge conflict on my side ;-)

Reply:
Well, I didn't see a point in having `refract or 0`. Because the value of this Boolean expression is always `refract`. That's why I removed the `or 0` part.